docs/API/V1/README.md

```diff
@@ -20,9 +20,9 @@ While logged in, you can retrieve (`GET`) and modify (`PUT`) your user profile t
 
 You can read a more complete specification of Pbench server metadata at [metadata](../metadata.md).
 
-When a dataset is first processed, the Pbench server will populate basic metadata, including the creation timestamp, the owner of the dataset (the user authorization token used to authenticate the `PUT` from the agent via `pbench-results-move`), and the full contents of the dataset's `metadata.log` file inside the dataset tarball. The Pbench server will also calculate a default deletion date for the dataset based on the owner's retention policy and the server administrator's retention policy.
+When a dataset is first processed, the Pbench server will populate basic metadata, including the creation timestamp, the owner of the dataset (the user associated with the authorization token given to the agent `pbench-results-move` command), and the full contents of the dataset's `metadata.log` file inside the dataset tarball. The Pbench server will also calculate a default deletion date for the dataset based on the owner's retention policy and the server administrator's retention policy.
```

Comment:
s/the Pbench server/the Pbench Server/
s/the agent/the Pbench Agent/

Also at line 27.